environment.yml

```diff
@@ -24,3 +24,5 @@ dependencies:
 - networkx
 # tests
 - pytest
+- pip:
+    - roocs-utils>=0.1.4
```

Comment:
I would only put conda packages here and keep the packages from pypi only in `requirements.txt`.

The reason for this is to create a conda spec file which can only have conda packages.

Reply:
Ok that makes sense, I'll take it out.

I'm waiting for the clisops import fix before I convert this back to an open PR